databases/week2/Part4.sql

```diff
@@ -0,0 +1,133 @@
+-- MySQL Script generated by MySQL Workbench
+-- Fri 16 Oct 2020 01:07:37 PM CEST
+-- Model: New Model    Version: 1.0
+-- MySQL Workbench Forward Engineering
+
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
```

Comment:
do you understand these syntex line7,8,9?

Reply:
I am afraid I do not! It has been added by workbench model  when I created EER model from database!

Reply:
@arya56 you can read a bit about it here: https://dev.mysql.com/doc/workbench/en/workbench-faq.html#faq-workbench-increase-performance